### PR TITLE
NXP-26900: Hide browsing by tags tab in JSF if faceted tag implementation

### PR DESCRIPTION
**Pull request by funshodavid**
T&P currently failing because on unrelated issue (NXP-27041)

### CODE REVIEW

**Comment by kevinleturc** [comment]
👍

File: nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-web/src/main/resources/OSGI-INF/navtree-contrib.xml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0"?>
 <component name="org.nuxeo.ecm.virtualnavigation.tagsContrib">
 
+  <require>org.nuxeo.ecm.platform.actions</require>
```
Comment (troger) [inline nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-web/src/main/resources/OSGI-INF/navtree-contrib.xml]: Is this really needed?

**Comment by funshodavid** [comment]
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/TestAndPush/job/ondemand-testandpush-fdavid-master/616/

File: nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-web/src/main/resources/OSGI-INF/navtree-contrib.xml (hunk 1 above)
Comment (funshodavid) [reply nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-web/src/main/resources/OSGI-INF/navtree-contrib.xml]: Maybe not, I'll try without

Comment (funshodavid) [reply nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-web/src/main/resources/OSGI-INF/navtree-contrib.xml]: Looks like the contributed action is not correctly merged if this require is not added.

**Comment by troger** [comment]
:+1:

**Pull request merged**